extractor/src/extractors/JsonnetJsonExtractor.ts

```diff
@@ -0,0 +1,61 @@
+import {Dependency, DependencyManagementFile} from "@depscloud/api/v1alpha/deps";
+import MatchConfig from "../matcher/MatchConfig";
+import Extractor from "./Extractor";
+import ExtractorFile from "./ExtractorFile";
+import Languages from "./Languages";
+
+const extract = (dependency: any,scopes:any):Dependency => {
+    return {
+        organization:"",
+        module:"",
+        versionConstraint: dependency.version,
+        scopes:[scopes],
+        name: parseName(dependency),
+    };
+}
+
+//The name should be formatted as {{ remote }}[#{{ subdir }}] where subdir
+const parseName = (dependency) => {
+    return `${dependency.source.git.remote}[#${dependency.source.git.subdir}]`;
```

Comment:
Let's remove the `[]`. Most URL libraries will support parsing with a fragment.

Reply:
Also, let's make the subdir the scope for the dependency instead of a fragment on the name.

Reply:
You mean , I should remove subdir from name  and place in the scopes array of the dependency ?